src/util/adb.js

```diff
@@ -312,7 +313,11 @@ export default class ADBUtils {
     }];
 
     if (!apkComponent) {
-      apkComponent = '.App';
+      if (defaultApkComponents[apk]) {
+        apkComponent = defaultApkComponents[apk];
+      } else {
+        apkComponent = '.App';
+      }
```

Comment:
Nitpick, not that I do look to this again I think that we could unconditionally set '.App' first and then override it if we do have a specific default apk component for that apk. I mean just the following small tweak:
```
if (!apkComponent) {
  apkComponent = '.App';
  if (defaultApkComponents[apk]) {
    apkComponent = defaultApkComponents[apk];
  }
}
```